api/v3/licensemaster_types.go

```diff
@@ -55,12 +55,12 @@ type LicenseMasterStatus struct {
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// LicenseMaster is the Schema for a Splunk Enterprise license manager.
+// LicenseMaster is the Schema for a Splunk Enterprise Obsolete license master.
```

Comment:
Should we match the terminology with the spec [here](https://github.com/splunk/splunk-operator/pull/756/files#diff-809254b55d0140ed411813756f78738cb5836f972a111952db1078caec2e9bc3R108)? i.e 'Splunk Enterprise Obsolete license manager'.

Reply:
Resolved in the latest commit.